core/src/main/java/org/apache/accumulo/core/client/mapreduce/lib/partition/RangePartitioner.java

```diff
@@ -91,8 +92,7 @@ private synchronized int getNumSubBins() {
   private synchronized Text[] getCutPoints() throws IOException {
     if (cutPointArray == null) {
       String cutFileName = conf.get(CUTFILE_KEY);
-      Path[] cf = org.apache.accumulo.core.clientImpl.mapreduce.lib.DistributedCacheHelper
-          .getLocalCacheFiles(conf);
+      Path[] cf = DistributedCacheHelper.getLocalCacheFiles(conf);
```

Comment:
Is `job.getLocalCacheFiles()` deprecated?

Reply:
No that one is good.  I guess its safe to call Job.getInstance() here?

Reply:
> I guess its safe to call Job.getInstance() here?

I'm not sure what that does. Can't we just pass the job context into this private method?

Reply:
We could but we would be doing the same thing in the method calling it.  It is called by the overridden getPartition() in Partitioner.  The only params from it are: `Text key, Writable value, int numPartitions`

Reply:
Oh, that sucks... the partitioner has no native concept of job context. However, the job's configuration is passed in by the job context when it is initialized, because it implements Configurable. So, we could call `job.getInstance(getConf(), "dummy")` if it doesn't cause too much side-effects, so we can access the job's convenience method to get the local cache files. If the implementation is trivial enough, or if instantiating a new dummy job has unwanted side-effects, we could probably also just parse the config item storing the paths to the local cache files ourselves.

Reply:
I am not sure... it looks like any call "Creates a new job" https://hadoop.apache.org/docs/r2.4.1/api/org/apache/hadoop/mapreduce/Job.html#getInstance()

Reply:
Yeah, but if it's just a container object that holds the config and provides methods, but doesn't actually do anything until you submit it, it's probably fine. We'd have to check the Hadoop 3 impl of Job.